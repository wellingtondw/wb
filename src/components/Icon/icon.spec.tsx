import { renderWithTheme } from "../../utils/tests/helper";

import { Icon } from ".";

describe("<Icon />", () => {
  it("should be able to render correctly", () => {
    const { container } = renderWithTheme(<Icon icon="car" />);
    expect(container.firstChild).toHaveClass("icon-car");
    expect(container.firstChild).toMatchSnapshot();
  });
});
