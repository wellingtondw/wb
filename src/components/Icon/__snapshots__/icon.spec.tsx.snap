// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Icon /> should be able to render correctly 1`] = `
<span
  class="icon-car"
  style="font-size: 4.0rem;"
/>
`;
